scss/components/_itemvehicleweaponsheet.scss

```diff
@@ -27,39 +42,32 @@
       background-color: $swffg-weapon-color;
     }
   }
-  
-  .firing-arcs {

```

Comment:
Were these just legacy values? Can't see them being used or removed in any templates in this PR.

Reply:
That was an incorrect change. Apparently I copied and pasted the wrong sheet. Let me get that fixed.